### PR TITLE
#90: StringAsUrl and UrlAsString

### PR DESCRIPTION
**Pull request by fabriciofx**
As per #90:

- `StringAsUrl` introduced
- `UrlAsString` introduced

### CODE REVIEW

**Comment by 0crat** [comment]
@fabriciofx I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/text/StringAsUrl.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+     * @param url The URL as Text
+     * @param encoding The encoding
+     */
+    public StringAsUrl(final Text url, final Charset encoding) {
+        this.source = url;
+        this.encoding = encoding;
+    }
+
+    @Override
+    public String asValue() throws Exception {
```
Comment (yegor256) [inline src/main/java/org/cactoos/text/StringAsUrl.java]: @fabriciofx would be better to remove `throws Exception` and catch it inside

File: src/main/java/org/cactoos/text/UrlAsString.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+     * @param url The URL as Text
+     * @param encoding The encoding
+     */
+    public UrlAsString(final Text url, final Charset encoding) {
+        this.source = url;
+        this.encoding = encoding;
+    }
+
+    @Override
+    public String asValue() throws Exception {
```
Comment (yegor256) [inline src/main/java/org/cactoos/text/UrlAsString.java]: @fabriciofx same here, let's catch it inside

**Comment by yegor256** [comment]
@fabriciofx let's get rid of exceptions here, see my comments above. Also, would be great if you format pull request descriptions as I'm doing it, for example: https://github.com/yegor256/cactoos/pull/99

File: src/main/java/org/cactoos/text/StringAsUrl.java (hunk 1 above)
Comment (fabriciofx) [inline src/main/java/org/cactoos/text/StringAsUrl.java]: @yegor256 Ok!

File: src/main/java/org/cactoos/text/UrlAsString.java (hunk 2 above)
Comment (fabriciofx) [inline src/main/java/org/cactoos/text/UrlAsString.java]: @yegor256 Ok!

**Comment by fabriciofx** [comment]
@yegor256 Ok. I'll do it. Thanks for you attention!

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**